source/whats-new.txt

```diff
@@ -28,6 +28,38 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+.. _version-2.24:
+
+What's New in 2.24
+------------------
+
+The 2.24 driver release includes the following new features:
+
+- Added `IMongoCollection.DistinctMany() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.IMongoCollection-1.DistinctMany.html>`__ method.
```

Comment:
S: Since the last few bullets in this list are complete sentences, I think it makes sense to update the ones that are sentence fragments

```suggestion
- Added the `IMongoCollection.DistinctMany() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.IMongoCollection-1.DistinctMany.html>`__ method.
```

Reply:
Applies to the other bullet points too